packages/admin/cms-admin/src/contentScope/Provider.tsx

```diff
@@ -116,13 +117,21 @@ export function useContentScope(): UseContentScopeApi {
 }
 
 export interface ContentScopeProviderProps {
-    defaultValue: ContentScope;
-    values: ContentScopeValues;
+    defaultValue?: ContentScope;
+    values?: ContentScopeValues;
     children: (p: { match: match<NonNullRecord<ContentScope>> }) => ReactNode;
     location?: ContentScopeLocation;
 }
 
 export function ContentScopeProvider({ children, defaultValue, values, location = defaultContentScopeLocation }: ContentScopeProviderProps) {
+    const user = useCurrentUser();
+    if (values === undefined) {
+        values = user.allowedContentScopes;
+        defaultValue = user.allowedContentScopes[0]?.scope;
+    }
+    if (!values || !defaultValue) {
```

Comment:
This would evaluate to true for a user without allowed content scopes, is this correct?